file/reader.go

```diff
@@ -36,6 +36,7 @@ func GetForKonnect(fileContent *Content, opt RenderConfig) (*utils.KongRawState,
 	// setup
 	builder.targetContent = fileContent
 	builder.currentState = opt.CurrentState
+	builder.kongVersion = opt.KongVersion
```

Comment:
To confirm, we just set this to avoid a segfault; it won't actually have a value? Looks like that's the case.

Reply:
The empty value for semver.Version is 0.0 so there won't be a segfault. This is to guard against the problem from cropping again if there comes a point that we care about the version of Kong Control-Plane in the Konnect world.